Classes/Public/Manager/Submanagers/OCTSubmanagerCalls.h

```diff
@@ -30,7 +36,7 @@
 /**
  * Call sessions that are active.
```

Comment:
Btw, I'm not sure about this `active` word in comment. You have `active` call state, as well as `paused`, `dialing`, etc. It may be confusing that this container has only active calls, but doesn't contain paused